tests/integration/zorder/test_zorder.py

```diff
@@ -0,0 +1,35 @@
+from tests.integration.base import DBTIntegrationTest, use_profile
+
+
+class TestZOrder(DBTIntegrationTest):
+    @property
+    def schema(self):
+        return "zorder"
+
+    @property
+    def models(self):
+        return "models"
+
+    @property
+    def project_config(self):
+        return {"config-version": 2, "vars": {"databricks_skip_optimize": "True"}}
```

Comment:
Seems like we only test skipping optimization?

Reply:
oh,. good catch. I added that temporarily and forgot to remove